source/audio/soundSplit.py

```diff
@@ -186,23 +128,42 @@ def on_session_created(self, new_session: AudioSession):
 			channelVolume.SetChannelVolume(0, self.leftNVDAVolume, None)
 			channelVolume.SetChannelVolume(1, self.rightNVDAVolume, None)
 
+	def onSessionTerminated(self, session: AudioSession) -> None:
+		pid = session.ProcessId
+		try:
+			channelVolume = session.channelAudioVolume()
+			channelCount = channelVolume.GetChannelCount()
+			if channelCount != 2:
+				log.warning(
+					f"Audio session for pid {pid} has {channelCount} channels instead of 2 - cannot set volume!"
+				)
+				return
+			channelVolume.SetChannelVolume(0, 1.0, None)
+			channelVolume.SetChannelVolume(1, 1.0, None)
+		except Exception:
+			log.exception(f"Could not restore channel volume of process {pid} upon exit.")
+
 
-def setSoundSplitState(state: SoundSplitState, initial: bool = False) -> dict:
-	applyToFuture = True
+def setSoundSplitState(state: SoundSplitState) -> dict:

```

Comment:
this appears to be an API breaking change.
This could be avoided by keeping the dummy parameter and logging an error when its used